src/containers/SnippetArea.jsx

```diff
@@ -1,12 +1,13 @@
-import React, { PropTypes } from 'react';
-import { connect } from 'react-redux';
 import axios from 'axios';
-import _ from 'lodash'
+import _ from 'lodash';
 import { CardText, Snackbar } from 'material-ui';
-import { browserHistory } from 'react-router'
+import React, { PropTypes } from 'react';
```

Comment:
I think they should be sorted by order of importance. React is the most important so it should go first.